packages/iTwinUI-react/src/core/Buttons/Button/Button.tsx

```diff
@@ -71,14 +71,9 @@ export const Button: ButtonComponent = React.forwardRef((props, ref) => {
   return (
     <Element
       ref={ref}
-      className={cx(
-        'iui-button',
-        `iui-${styleType}`,
-        {
-          [`iui-${size}`]: !!size,
-        },
-        className,
-      )}
+      className={cx('iui-button', className)}
+      data-iui-variant={styleType}
+      data-iui-size={size || undefined}
```

Comment:
I think we don't want to add a data attribute for default variant and size, so we might need a ternary conditional here

Reply:
Size doesn't have `default` value. It's small or large. 
![image](https://user-images.githubusercontent.com/83585998/181451978-f9fe6037-49fa-470b-b02c-fbdc4b032261.png)

Added condition to variant data attribute.